anago

```diff
@@ -357,6 +357,14 @@ check_prerequisites () {
   for rb in ${WRITE_RELEASE_BUCKETS[*]}; do
     release::gcs::ensure_release_bucket $rb || return 1
   done
+
+  # Check for manifest-tool, if not install it
+  if [[ ! -f $(which manifest-tool) ]]; then
+    MANIFEST_TOOL_PATH=${MANIFEST_TOOL_PATH:-$(mktemp -d -t manifest)}
+    PATH=$MANIFEST_TOOL_PATH:$PATH
```

Comment:
@ixdy @david-mcmahon Where do you want to put this? Globally in e.g. `/usr/local/bin` if not present or in temp or whatever?

An alternative would be to use download the latest docker 18.02 client binary and use that for "official" pushing

Reply:
I'd prefer this to instruct the user to Do The Right Thing and install it somewhere permanent.

Reply:
+1 to David's comment

Reply:
I've already included this tool in `Dockerfile.k8s-cloud-builder` file and will be used if we use cloud-builder. Above flow is only for people who are trying it out locally. Having said that I'll add an instruction to install the tool and rerun the script if we don't find a tool in the path.

@david-mcmahon @jdumars @luxas @ixdy Does is it make sense?

Reply:
Yes, just look at the way the other prerequisites check and notify on dependencies and follow suit.  There is still local usage of the tooling so we want to get that right.

Reply:
@david-mcmahon @jdumars fixed it, PTAL latest commit. -Thanks